### DISCUSSION TITLE
Billing and Self Hosting

### DISCUSSION THREAD

**Discussion by gitfool** [chdsbd/kodiak#648] [Q&A] [answered]
I'm looking at using Kodiak with my personal projects but also with an eye towards advocating for it within organization projects.

I would be using the free option for personal projects, but with the above in mind, and due to price sensitivity outside of my control, I need to clarify what the intersection of [billing](https://kodiakhq.com/docs/billing) and [self hosting](https://kodiakhq.com/docs/self-hosting) is for private organization repositories - is that a free option too?

**Comment by chdsbd** [comment, marked as answer]
There is no intersection between billing and self hosting.

Self hosting is free. The [self hosting instructions](https://kodiakhq.com/docs/self-hosting) will help you run the Kodiak Docker image on Heroku, but those instructions should be easily adapted for other container platforms

The [hosted GitHub App](https://github.com/marketplace/kodiakhq) is free for public repositories and personal GitHub accounts.  For private GitHub Organization repositories you need to purchase seat licenses after the 30 day free trial.
